### PR TITLE
Uml 2334 : Make it clearer how to ask for a new activation key if given has expired

### PR DESCRIPTION
**Pull request by SeemaMenon**
# Purpose

https://opgtransform.atlassian.net/browse/UML-2334

Fixes UML-2334

## Approach

_Explain how your code addresses the purpose of the change_

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [X] I have performed a self-review of my own code
* [X] I have added relevant logging with appropriate levels to my code
* [x] New event_codes have been documented on the [wiki page](https://opgtransform.atlassian.net/wiki/spaces/LSML2/pages/3277881441/Understanding+the+event+logs)
* ~I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant~
* [X] I have added tests to prove my work
* [x] I have added welsh translation tags and updated translation files
* [X] I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* [x] I have notified the Interaction Designer of any content changes so that appropriate screenshots/flow diagram changes can be made
* [x] The product team have tested these changes

### CODE REVIEW

**Comment by SeemaMenon** [comment]
Screenshot 2022-07-28 at 13 18 53

**Comment by SeemaMenon** [comment]
Screenshot 2022-07-29 at 10 29 31
Screenshot 2022-07-29 at 10 29 58

File: service-front/app/src/Actor/templates/actor/add-lpa-triage.html.twig [hunk 1]
```diff
@@ -15,7 +15,7 @@
         <main class="govuk-main-wrapper" id="main-content" role="main">
             <div class="govuk-grid-row">
                 {{ govuk_form_open(form) }}
-                <div class="govuk-grid-column-two-thirds">
+                <div class="govuk-grid-column-three-quarters">
```
Comment (lukejolliffe) [inline service-front/app/src/Actor/templates/actor/add-lpa-triage.html.twig:18]: Unsure why this needs to change to three-quarters. Preferably I would want to stick to the GDS suggested two thirds as much as possible

Comment (lukejolliffe) [inline service-front/app/src/Actor/templates/actor/add-lpa-triage.html.twig:18]: Unsure why this needs to change to three-quarters. Preferably I would want to stick to the GDS suggested two thirds as much as possible

**Review by lukejolliffe — CHANGES_REQUESTED** [review]

File: service-front/app/src/Actor/templates/actor/add-lpa-triage.html.twig (hunk 1 above)
Comment (SeemaMenon) [reply service-front/app/src/Actor/templates/actor/add-lpa-triage.html.twig]: Reverted

**Review by lukejolliffe — APPROVED** [review]

**Pull request merged**